indexer/src/main/java/org/pdxfinder/dataloaders/updog/tablevalidation/rules/PdxValidationRuleset.java

```diff
@@ -66,12 +73,12 @@ public TableSetSpecification generate(String provider) {
                     ColumnReference.of("metadata-model.tsv", "model_id"),
                     ColumnReference.of("metadata-sharing.tsv", "model_id")),
                 Relation.betweenTableColumns(
-                        Relation.validityType.one_to_many,
+                        validityType.ONE_TO_MANY,
```

Comment:
Could we capitalize the validityType enum?

Reply:
That is odd.

Reply:
Definitely.